guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Lightning service" icon="info" color="blue" %}
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+With Lightning services, the user can fund their wallet directly with a Lightning payment saving time and costs.
 
 {% include /tip/close.html %}
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+### Network sync
 
+If your application uses a method like compact block filters (BIP157/158) to sync to the Bitcoin network, which may take some time to sync, communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete to prevent user confusion that their funds have not shown up.
 
-{% include fact/cons.html %}
- - Managing taxes
- - Difficult to find businesses/individuals willing to pay in bitcoin
- - Hard to set up recurring payments
- - Not private
- {% include fact/close.html %}
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
 
+## Funding the wallet
 
-### Redeem bitcoin using a gift card
+The next step for the user to take is sending bitcoin to the payment request.
 
-Gift cards allow users to buy and own small amounts of bitcoin without having to sign up for an exchange. These cards are sometimes available from local retailers.
+Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This avoids the user having to manually find a peer and open a channel.
 
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
+
+Once a payment channel is open, the user's wallet is ready to use. As the user's newly opened channel only has outbound liquidity they will not be able to receive Lightning payments straight away.
```

Comment:
> As the user's newly opened channel only has outbound liquidity they will not be able to receive Lightning payments straight away.

I think this is a pain-point we can try to offer solution to. For manual channel opening this is okay to happen, but if an LSP opens channel to us?

Breez resolves this by opening % on top of the amount, but again this is risk a wallet provider may or may not be wiling to make.

Reply:
So I aimed to cover using LSPs in a sub-page of the funding a wallet page whereas this flow uses none. I added the tips to point towards a solution for this (these will have links to the sub-page once it is live).